Adding iiQKA.OS2 support (#228)
kroshu/kuka_drivers: doc/wiki/3_iiQKA.OS2_RSI.md

## iiQka driver (RSI)

### Setup

#### Client side
It is recommended to use the driver on a real-time capable client machine (further information about setting up the PREEMPT_RT patch can be found [here](https://github.com/kroshu/kuka_drivers/wiki/6_Realtime)).

To set up the controller with iiQWorks.Sim (which is necessary if RSI is not yet installed), a Windows machine is also required.

##### Client IP configuration

> - Use the KRC's built in DHCP server (KSI) to connect to the windows machine (in this case leave the subnet settings as default) or set a fixed IP in the subnet of the KLI interface (in this case you can not have connection with both PCs). This is required to connect to iiQWorks.Sim and transfer the project to the KRC

Reviewer: you can have connections with multiple PC-s using a switch, so I would delete that comment

#### Controller side
These instructions were tested with RSI 6.0.0 (on iiQKA.OS2)

##### Controller network configuration

RSI can only communicate via the KLI network. Make sure that the controller network and the **PC with ROS** is connected to the same subnet.

1. Log in as **Expert**, **Safety Maintenance Technician** or **Administrator** on the smartped and in the main menu bar navigate to **Network** menu (**System > System settings > Network**).

2. Change network settings:
    - Set the **Allocation** to manual, to connect the ROS PC directly with the controller
    - Set the **IP address** and the **Subnet mask** to the chosen address

3. After pressing **Apply** the new network address should be configured.

##### RSI configuration
Configure your project in iiQWorks.Sim with the following steps:

1. Add the desired RSI version (6.0.0 or above) to iiQWorks.Sim
    - Download RSI option package from the **Software Repository** in **iiQworks.Cockpit**
    - Open the Option Package Manager by the manage button in the **Add-ons** (**File > Options > Add-on > KUKA Option packages > Manage**)

>     - By clicking on the **plus sing** add the downloaded option package from your file system (it should be a **.kop** file).

Reviewer: ```suggestion
    - By clicking on the **plus sign** add the downloaded option package from your file system (it should be a **.kop** file).
```

2. After adding the desired robot from the eCatalog by dragging it to the layout configure the option packages to use.
    - In the **Component properties**, under **Option packages configuration** add the desired option packages with the plus sign.

> 3. After adding the RSI option package under the **Home** page in the **devices** tab under **Option packages > iiQka.RobotSensorInterface** there is two menu available.

Reviewer: ```suggestion
3. After adding the RSI option package under the **Home** page in the **devices** tab under **Option packages > iiQka.RobotSensorInterface**, there are two menu options available:
```

    - The first one is a list of available context files. You can import, export and create new context files. The files can be visualized and edited with **RSIVisual**.

>     - The second is a list of ethernet congiuration files. You can import, export and create new context files. The files can be visualized and edited with **XML editor**.

Reviewer: ```suggestion
    - The second is a list of ethernet configuration files. You can import, export and create new context files. The files can be visualized and edited with **XML editor**.
```
